pkg/cmd/server/admin/create_nodeconfig.go

```diff
@@ -395,7 +395,17 @@ func (o CreateNodeConfigOptions) MakeNodeConfig(serverCertFile, serverKeyFile, n
 		return err
 	}
 
-	content, err := latestconfigapi.WriteYAML(config)
+	// Roundtrip the config to v1 and back to ensure proper defaults are set.
```

Comment:
https://github.com/openshift/origin/issues/2591  @liggitt please assign to me or to @stevekuznetsov